packages/common/components/content/blocks/item.marko

```diff
@@ -30,6 +30,21 @@ $ const imageLinkAttrs = { title: content.shortName, ...input.imageLinkAttrs };
   <@title tag="h5" modifiers=input.titleModifiers>
     <cms-content-short-name tag=null obj=content link=true />
   </@title>
+  <if(input.withContactTitle)>
```

Comment:
I would prefer this to be a separate contact-item component that just uses the regular item under the hood. That way the current item component doesn’t get so cluttered